### DISCUSSION TITLE
Is it possible to encode a text file back to the same format as VBAProject.bin file.

### DISCUSSION THREAD

**Discussion by Ofrench** [decalage2/oletools#824] [Ideas]
Currently using oletools to extract macros from an excel sheet to put in source control. I'm wondering if it's possible to make changes to output text file, convert it back to a binary file and then replace the VBAProject.bin file with new version. For example if I changed the code in the attached text file to write "Workbook closed" instead of "Workbook exit". I would want to encode that to a binary file and then replace the macro in the excel book. 

Thanks!

**Comment by decalage2** [comment]
Hi, this is technically possible as the compression algorithm is documented by Microsoft, but it has not been implemented in oletools so far. 
This has also been discussed in #555 with some alternative suggestions.

**Reply by Ofrench** [reply]
Thanks so much!
